src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -226,14 +212,14 @@ private boolean isActionPlanLive(ActionCase actionCase) {
         && actionCase.getActionPlanEndDate().after(currentTime);
   }
 
-  private void createActionsForCaseAndRule(ActionCase actionCase, ActionRule actionRule) {
+  private void createActionForCaseAndRule(ActionCase actionCase, ActionRule actionRule) {
     if (hasRuleTriggered(actionRule)) {
       try {
-        createActions(actionCase, actionRule);
+        createAction(actionCase, actionRule);
       } catch (Exception ex) {
         log.with("case_id", actionCase.getId().toString())
             .with("action_rule_id", actionRule.getId().toString())
-            .error("Failed to create actions", ex);
+            .error("Failed to create action", ex);
```

Comment:
I'm a bit uncomfortable about logging errors and not rethrowing them - what if the exception was due to a temporary blip contacting Postgres? We'd want the whole transaction rolled back so that we could retry, wouldn't we?